libraries/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -622,6 +660,17 @@ export class RushConfiguration {
 
     this.ensureConsistentVersions = !!rushConfigurationJson.ensureConsistentVersions;
 
+    // Check if we have a subspace configuration file
+    const subspaceConfigLocation: string = path.join(
```

Comment:
This path is already generated inside `SubspaceConfiguration.loadFromDefaultLocation`. Just call that with `this` as an argument instead of generating this path again.

Reply:
changed